command-expansion-server/build.gradle

```diff
@@ -0,0 +1,19 @@
+
+plugins {
+  id "com.github.node-gradle.node" version "3.1.1"
+}
+
+node {
+  version = '16.4.0'
+  npmVersion = '8.5.0'
+  download = true
+}
+
+task build(type: NpmTask) {
+  dependsOn npmInstall
+  args = ['run','build']
+}
+
+task clean(type: Delete) {
+  delete '.gradle', 'build','node_modules', 'package-lock.json'
```

Comment:
I think you can safely remove `'.gradle'` from this list here. `.gradle/` should only contain cached Gradle dependencies, and in this Gradle subproject the only cached dep. should be the `node-gradle` plugin.

Reply:
I also think `'build'` can be removed since this task (`clean`) is not overriding the default Gradle `clean` task (which removed `build/`). I'd double check me on this though.

Reply:
Thanks for the suggestion. I should make a task called `nuke` which will remove everything. :P

Reply:
I just verified and `'build'` is needed. `.gradle clean` doesn't remove the `build/` directory in the javascript project.

Reply:
Thanks for checking! :)